test/installation-proxy/installation-proxy-service-specs.js

```diff
@@ -0,0 +1,29 @@
+import chai from 'chai';
+import InstallationServiceProxy from '../../lib/installation-proxy';
+import PlistService from '../../lib/plist-service';
+import { getServerWithFixtures, fixtures } from '../fixtures';
+
+
+chai.should();
+
+describe('installation proxy', function () {
+  let server;
+  let socket;
+  let installationServiceProxy;
+
+  afterEach(function () {
+    if (installationServiceProxy) {
```

Comment:
does it make sense to assign vars back to null after they are closed or multiple close call on the same instance is safe?

Reply:
I don't like doing to because it causes null pointer issues.

Reply:
like I said if you don't then there is a possibility that close can be called twice on the same instance

Reply:
That is fine thou. Usually,  resource releasing methods are idempotent. I tested it with this one as well, it is all good